ubuntu_mint.sh

```diff
@@ -74,11 +78,24 @@ function nuc_data_make {
 }
 
 function test_pyne {
+    
+    # only test for python version if using the most recent dev branch
+    if [ $1 == 'dev' ] ; then
+    
+        # check which python version to run correct tests
+        version=`python -c 'import sys; print(sys.version_info[:][0])'`
+
+        # Run all the tests
+        cd tests
+        if [ $version == '2' ] ; then
+            source ./travis-run-tests.sh python2
+        elif [ $version == '3' ] ; then
+            source ./travis-run-tests.sh python3
+        fi
 
-    # Run all the tests
-    cd tests
-    source ./travis-run-tests.sh
-
+    elif [ $1 == 'stable' ] ; then
+        source ./travis-run-tests.sh
```

Comment:
Don't we need to change to the tests directory first?  Maybe line 89 needs to come before line 83?

Reply:
yes, need to fix that